In [1]:
import pandas as pd 
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
from scipy import stats

## Q.1) 1.Examine the following normal Quantile plots carefully. Which of these plots indicates that the data …

## Ans:
#### i) C
#### ii) B
#### iii) A,C,D
#### iv) A

## Q.2) For each of the following statements, indicate whether it is True/False. If false, explain why.

### The manager of a warehouse monitors the volume of shipments made by the delivery team. 
### The automated tracking system tracks every package as it moves through the facility. 
### A sample of 25 packages is selected and weighed every day. Based on current contracts with customers, 
### the weights should have μ = 22 lbs. and σ = 5 lbs.

### Before using a normal model for the sampling distribution of the average package weights, 
### the manager must confirm that weights of individual packages are normally distributed.

### The standard error of the daily average SE(x ̅) = 1.



## Ans:
#### i) True : The individual value do not need to be normally distributed as long as sample size condition satisfied.
#### By CLT condition n>=30. In this case samlpe size is not satisfing CLT condition,hence individual checking is required.

#### ii) True: S.E=sigma/sqrt(n)=5/sqrt(25)=5/5=1

## Q.3) Auditors at a small community bank randomly sample 100 withdrawal transactions made during the week at an ATM machine located near the bank’s main branch. Over the past 2 years, the average withdrawal amount has been 50 with a standard deviation of 40. Since audit investigations are typically expensive, the auditors decide to notinitiate further investigations if the mean transaction amount of the sample is between 45 and 55. What is the probability that in any given week, there will be an investigation?

### A.	1.25%
### B.	2.5%
### C.	10.55%
### D.	21.1%
### E.	50%



## Ans:
#### Z=X-mu/(sigma/sqrt(n))
#### Z(45)=(45-50)?(40/sqrt(100))
#### z(45)=-1.25  similarlly
#### Z(55)=1.25


In [3]:
stats.norm.cdf(1.25)-stats.norm.cdf(-1.25)

0.7887004526662893

#### Probability for do not investigate is 0.7887004526662893
#### Probability for investigation = 1-0.7887004526662893 = 0.2113
#### i.e 21.13% , hence Ans is D

## Q.4) The auditors from the above example would like to maintain the probability of investigation to 5%.
## Which of the following represents the minimum number transactions that they should sample if they do not want to 
## change the thresholds of 45 and 55? Assume that the sample statistics remain unchanged.

### A.	144
### B.	150
### C.	196
### D.	250
### E.	Not enough information



## Ans:
#### Z stats for 95% confidence interval is 1.96
#### Z=X-mu/(sigma/sqrt(n))
#### Z=(55-50)(sqrt(n))/40
#### sqrt(n)=1.96(40)/5 = 15.68
#### n=(15.68)^2
#### n=245.86 ~ 246
#### Ans is D.

## Q.5) An educational startup that helps MBA aspirants write their essays is targeting individuals who have taken GMAT in 2012 and have expressed interest in applying to FT top 20 b-schools. There are 40000 such individuals with an average GMAT score of 720 and a standard deviation of 120. The scores are distributed between 650 and 790 with a very long and thin tail towards the higher end resulting in substantial skewness. Which of the following is likely to be true for randomly  chosen samples of aspirants?

### A.	The standard deviation of the scores within any sample will be 120.
### B.	The standard deviation of the mean of across several samples will be 120.
### C.	The mean score in any sample will be 720.
### D.	The average of the mean across several samples will be 720.
### E.	The standard deviation of the mean across several samples will be 0.60


## Ans:
#### SEM=sigma/sqrt(n)
#### SEM=120/sqrt(40000) = 120/20 = 0.60
#### Ans is E.